collect_app/src/main/java/org/odk/collect/android/formentry/FormIndexAnimationHandler.java

```diff
@@ -10,6 +10,11 @@
  */
 public class FormIndexAnimationHandler {
```

Comment:
I think I managed to take this another step in the right direction. A lot of work to do still though!